tests/acceptance/helpscount-beacon-test.js

```
@@ -0,0 +1,49 @@
+
+    signInAsAdmin(this.owner, this.server);
+    await catalogPage.visit();
+
+    // await pauseTest();
+    await catalogPage.header.clickOnHeaderLink('Badges');
+
+    await animationsSettled();
+
+    // await waitUntil(() => window.Beacon, { timeout: 1000 });
+
+    assert.ok(!window.Beacon, 'Helpscout Beacon is not initialized');
```

ESLint

[error] Unexpected negation in assert.ok() assertion.

(qunit/no-negated-ok)